examples/save_to_file/example.exs

```diff
@@ -217,12 +217,12 @@ defmodule Peer do
   end
 end
 
-{:ok, pid} = Peer.start_link()
+{:ok, pid} = Peer.start()
 ref = Process.monitor(pid)
 
 receive do
-  {:DOWN, ^ref, _, _, _} ->
-    Logger.info("Peer process closed. Exiting")
+  {:DOWN, ^ref, _, _, reason} ->
+    Logger.info("Peer process closed, reason: #{inspect(reason)}. Exiting")
```

Comment:
I wonder if we should rename `Peer` module to `Client`. Right now we might get the following log message:

```
21:26:32.659 [info] Received unexpected message: %{"type" => "peer_left"}

21:26:32.665 [info] Peer process closed, reason: {:shutdown, :peer_closed_for_writing}. Exiting
```

which might be misleading as we don't know WHICH peer closed for writing :thinking: 
On the other hand, I am not sure if `Client` fits well :thinking:

Reply:
I don't think it's that big of a deal. Anyone running the example will most likely look into the its code and see where this messages is logged, so I don't think that's gonna be too confusing.